cloud-controller-manager/cmd/digitalocean-cloud-controller-manager/main.go

```diff
@@ -23,14 +23,21 @@ import (
 	"k8s.io/component-base/logs"
 	_ "k8s.io/component-base/metrics/prometheus/clientgo" // load all the prometheus client-go plugins
 	_ "k8s.io/component-base/metrics/prometheus/version"  // for version metric registration
-	"k8s.io/kubernetes/cmd/cloud-controller-manager/app"
+	"k8s.io/cloud-provider/app"
+	"k8s.io/cloud-provider/options"
+	"k8s.io/cloud-provider/app/config"
 
 	"github.com/digitalocean/digitalocean-cloud-controller-manager/cloud-controller-manager/do"
 	"github.com/spf13/pflag"
 )
 
 func main() {
-	command := app.NewCloudControllerManagerCommand()
+	opts, err := options.NewCloudControllerManagerOptions()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to construct options: %v\n", err)
```

Comment:
Is it correct that we emit an error message but do not terminate the program?

Reply:
Oops, good catch. Fixed.